sale_invoice_policy/pyproject.toml

```diff
@@ -1,3 +1,9 @@
 [build-system]
 requires = ["whool"]
 build-backend = "whool.buildapi"
+[project]
+name = "odoo-addons-oca-base_partition"
```

Comment:
@therbin200 You should not do that.

To include open PR's dependencies, you should follow: https://github.com/OCA/maintainer-tools/wiki/Use-temporary-reference%28s%29-to-another-pull-request%28s%29

Thanks